test/SerialIOTest.cpp

```diff
@@ -1451,9 +1443,11 @@ TEST_CASE( "adios1_fileBased_write_test", "[serial][adios1]" )
     {
         *positionOffset_local_1 = positionOffset_global[i];
         e_1["positionOffset"]["x"].storeChunk({i}, {1}, positionOffset_local_1);
-        o.flush();
     }
 
+    //TODO breaks openWriteFile logic when removed
+    //o.flush();
```

Comment:
works now?

Reply:
No, unfortunately. I figured out where it all goes haywire, but not what causes the behavior.
It has to do with ADIOS's distinction between read/write files. The scenario under which I can reproduce this is when creating a ```fileBased``` Series and *not* flushing between creation of two Iterations (i.e. files). The problem then becomes that the file with the earliest creation time will have an on-disk size of 0 Bytes and only the last file will be created as expected.
I doubt this is a problem with ADIOS itself, but the way the backend is using it:
File creation (i.e. opening a write file) is deferred until required (i.e. encountering a Dataset write operation). Once this file is opened, it will not be closed until new defines or reads happen.
... and this is exactly where the problem happens. One of these calls happens at the wrong time (for ADIOS), so the file on disk will be empty.
Unfortunately, ADIOS doesn't fail (only warns) when opening such a file. The SIGSEV in the test happens when that faulty file is closed.

Reply:
(Likely) found the culprit:
In ```fileBased``` Series, *one* ADIOS group was used for *all* files. This works when only one file is open at a time, but that is not a constraint enforced in the ADIOS backends.
A solution is to create a group for every file handle. The commit eliviates the CI failure, but does not reliably use the group across the backend.